dnf-behave-tests/features/repoquery/deps.feature

```diff
@@ -79,13 +79,14 @@ Scenario: repoquery --requires --resolve --recursive NAME-VERSION
       """
 
 Scenario: repoquery --requires --resolve --recursive --tree NAME-VERSION
+ When I execute dnf with args "repoquery --requires --resolve --recursive --tree top1-2.0"
  When I execute dnf with args "repoquery --requires --resolve --recursive --tree top1-2.0"
```

Comment:
This doubled When execution is intended or a forgotten patch development residue?

Reply:
Ah, sorry... forgot to remove this. It's gone now.